ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -34,9 +33,10 @@ message Commands {
   // Required
   string application_id = 3;
 
-  // Uniquely identified the command. This identifier should be unique for each new command within an
-  // application domain, i.e., the triple (application_id, party, command_id) must be unique.
-  // It can be used for matching the requests with their respective completions.
+  // Uniquely identifies the command.
+  // The triple (application_id, party + act_as, command_id) constitutes the change ID for the intended ledger change,
+  // where party + act_as is interpreted as a set of party names.
+  // The change ID can be used for matching the intended ledger changes with their respective completions.
```

Comment:
Isn't this what the `submission_id` is good for? Since there can be multiple completions for the same change ID (e.g., if there's a completion that says that the command already exists).

Reply:
You're right, slightly changed the wording to emphasize the change ID can be used for finding all relevant completions.

Reply:
Addressed in #10851.